### PR TITLE
[Dependency] Remove vbguest

### PR DESCRIPTION
**Pull request by cjreed121**
### Please check if the PR fulfills these requirements:

* [x] Documentation has been updated/added if relevant

### What is the current behavior?
Currently we use the vbguest plugin in vagrant. This is no longer necessary and creating issues while installing Ubuntu 20.04.

### What is the new behavior?
This PR removes the check for requiring vbguest.

### Other information?
To test make sure you remove vbguest from your system.
This can be done by doing: `vagrant plugin uninstall vagrant-vbguest`
Docs will also be updated to remove the step saying this is a requirement.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -38,10 +38,6 @@ VERSION=$(lsb_release -sr | tr '[:upper:]' '[:lower:]')
 bash ${GIT_PATH}/.setup/vagrant/setup_vagrant.sh #{extra_command} 2>&1 | tee ${GIT_PATH}/.vagrant/install_${DISTRO}_${VERSION}.log
 SCRIPT
 
-unless Vagrant.has_plugin?('vagrant-vbguest') || ENV.has_key?('SKIP_VBGUEST')
```
Comment (MasterOdin) [inline Vagrantfile:41]: We can remove usage of the `SKIP_VBGUEST` flag from the vagrant CI job.

**Review by MasterOdin — CHANGES_REQUESTED** [review]
Have you manually triggered the Vagrant CI action to confirm that that runs through?

**Comment by cjreed121** [comment]
Just started them now [https://github.com/Submitty/Submitty/actions/runs/964612347](https://github.com/Submitty/Submitty/actions/runs/964612347). Will comment here the results.

**Comment by cjreed121** [comment]
Both 18.04 and 20.04 passed on the Vagrant CI action.

**Comment by cjreed121** [comment]
Documentation updated at https://github.com/Submitty/submitty.github.io/pull/351

**Review by MasterOdin — APPROVED** [review]

**Pull request merged**